### DISCUSSION TITLE
Downloading extension behind proxy

### DISCUSSION THREAD

**Discussion by arunswarnam** [eclipse-theia/theia#11120] [Q&A] [answered]
Hi,
 We are using @theia/cli: 1.19.0. I notice `theia download:plugins` fails to download the extension's dependencies when run behind a proxy. For example attempting to download https://open-vsx.org/extension/42Crunch/vscode-openapi/4.7.1 behind proxy using @theia/cli fails to fetch the extension's [dependency](https://github.com/42Crunch/vscode-openapi/blob/v4.7.1/package.json#L30) 
![image](https://user-images.githubusercontent.com/10571107/167480336-10e445ef-3581-484b-bab6-221736f78972.png). Is there a known workaround for this or am I missing something?

**Comment by vince-fugnitto** [comment, marked as answer]
> Is there a known workaround for this or am I missing something?

@arunswarnam thank you for the discussion, there is an issue tracking the request already https://github.com/eclipse-theia/theia/issues/10267.

**Reply by arunswarnam** [reply]
Thank you! Will keep track of the issue.
